_config_automation.yml

```diff
@@ -14,51 +14,71 @@ data_dest: github
 folder_path:
 
 ###### Calendly ######
-refresh-calendly: yes
+refresh-calendly: no
 calendly_googlesheet:
 
 ###### Citations ######
 refresh-citations: yes
 citation_papers: [
-  "https://scholar.google.com/scholar?cites=6140457238337460780",
-  "https://scholar.google.com/scholar?cites=15945159957067506879" ]
+  "https://scholar.google.com/scholar?cites=9780221468138198188",  #tandfonline paper
+  "https://scholar.google.com/scholar?cites=3064856349813099026" ] #f1000 paper
 citation_googlesheet:
 
 ###### CRAN ######
 refresh-cran: yes
-cran_packages: [ metricminer, ottrpal ]
+cran_packages: [ OCSdata ]
 cran_googlesheet:
 
 ###### GitHub ######
 refresh-github: yes
 github_repos: [
-  fhdsl/metricminer,
-  fhdsl/metricminer-dashboard,
-  jhudsl/OTTR_Template,
-  jhudsl/OTTR_Template_Website
+  opencasestudies/ocs-bp-school-shootings-dashboard, #in OCS Search table
+  opencasestudies/ocs-bp-youth-disconnection, #in OCS Search table
+  opencasestudies/ocs-bp-opioid-rural-urban, #in OCS Search table
+  opencasestudies/ocs-bp-vaping-case-study, #in OCS Search table
+  opencasestudies/ocs-bp-youth-mental-health, #in OCS Search table
+  opencasestudies/ocs-bp-rural-and-urban-obesity, #in OCS Search table
+  opencasestudies/ocs-bp-RTC-wrangling, #in OCS Search table
+  opencasestudies/ocs-bp-RTC-analysis, #in OCS Search table
+  opencasestudies/ocs-bp-co2-emissions, #in OCS Search table
+  opencasestudies/ocs-bp-diet, #in OCS Search table
+  opencasestudies/ocs-bp-air-pollution, #in OCS Search table
+  opencasestudies/ocs-healthexpenditure, #in OCS Search table
+  opencasestudies/OCS_Guide, #Guide for using OCS in teaching
+  opencasestudies/ocs-template, #OCS template repo (this one or the bp one?)
+  opencasestudies/ocs-bp-template #OCS template repo (this one or the non-bp one?)
```

Comment:
this is the template for making bp branded case studies

Reply:
It looked like this is the one the youtube video pointed to for using a template to make your own case study. So should I track both templates or just this one? I also will use the data from them differently (template and guide different from published case study) since they have different use cases.

Reply:
hmm yeah the other one would be kinda weird - it is mostly for internal use